src/group_theory/subgroup.lean

```diff
@@ -2311,6 +2311,10 @@ S.to_submonoid.has_faithful_scalar
 instance [add_monoid α] [distrib_mul_action G α] (S : subgroup G) : distrib_mul_action S α :=
 S.to_submonoid.distrib_mul_action
 
+/-- The action by a subgroup is the action by the underlying group. -/
```

Comment:
```suggestion
/-- The action by a subgroup is the action by the ambient group. -/
```

Reply:
This docstring is copied from lots of other places, so I'd prefer to leave it as is rather than finding all occurences in this PR.